modules/learn/pages/data/transactions.adoc

```diff
@@ -5,38 +5,23 @@
-Only following commit of the documents modified by the transaction &#8212; when the modified data-copy is removed from the meta data area, and is written over the main body of the document’s data &#8212; do the corresponding changes become readable by other transactions and operations.
-This isolation-level is known as https://jepsen.io/consistency/models/read-committed[Read Committed].
-Note that this use of a document’s extended attributes area results in an approximate doubling of its pre-transaction size, while the transaction is underway.
-In consequence, transactions can only be used on documents whose maximum size is 10 MB (which is half of the maximum-permitted document-size, 20 MB).
+Couchbase provides Distributed ACID Transactions for K-V (data) and Query operations -- these are available through APIs in 
+xref:cxx-txns::distributed-acid-transactions-from-the-sdk.adoc[C++], 
+xref:dotnet-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[.NET], and 
+xref:java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[Java].
 
-As well as making use of individual documents' extended attributes, transactions also create additional documents in each bucket that they access.
-These include:
 
-*  _Active Transaction Records_.
-Multiple Active Transaction Records can exist per bucket.
-Each of these documents can contain entries for multiple transactions.
-Collectively, these documents record all currently active transactions for the bucket.
-The name of each document is prefixed with `&#95;txn:atr-`.
+== Understanding Transactions
 
-* _Transaction Client Records_.
-One of these exists per bucket, covering all clients currently accessing the bucket transactionally.
-The document is named `&#95;txn:client-record`.
+_Distributed ACID Transactions_ are operations that ensure that when multiple documents need to be modified such that only the successful modification of all justifies the modification of any, either all the modifications do occur successfully; or none of them occurs.
+This _Atomicity_ supports _insert_, _update_, and _delete_ operations, across any number of documents.
+It also enables seemless integration between KV and xref:n1ql:n1ql-language-reference/begin-transaction.adoc#purpose[Query] operations within transactions.
 
-These documents, which may persist indefinitely, are automatically maintained by Couchbase Server, and should not be modified by any application.
+Only following commit of the documents modified by the transaction -- when the modified data-copy is removed from the meta data area (which uses xref:learn:data/extended-attributes-fundamentals.adoc[Extended Attributes]), and is written over the main body of the document’s data -- do the corresponding changes become readable by other transactions and operations.
```

Comment:
Well, other transactions can read based on MAV.  I'm not sure how much we want to cover that here, admittedly - maybe one for PM.

Reply:
Acknowledged.